packages/client-common/src/data_formatter/format_query_params.ts

```diff
@@ -51,6 +54,21 @@ export function formatQueryParams(
       : `${unixTimestamp}.${milliseconds.toString().padStart(3, '0')}`
   }
 
+  if (value instanceof TupleParam) {
```

Comment:
`Array` seems to have the same format. Can we accept JS `Array` as a valid data structure for `Tuple`? It would improve DX since no special class `TupleParam` is needed

Reply:
Array is `[A,B,...]`, not `(A,B,...)`

```
curl "http://localhost:8123?param_val=('foo','bar')" --data-binary "SELECT {val:Array(String)} JSONEachRow"
Code: 130. DB::Exception: Array does not start with '[' character: value ('foo','bar') cannot be parsed as Array(String) for query parameter 'val'. (CANNOT_READ_ARRAY_FROM_TEXT) (version 24.8.3.59 (official build))
```

```
curl "http://localhost:8123?param_val=\['foo','bar'\]" --data-binary "SELECT {val:Array(String)} FORMATJSONEachRow"
['foo','bar']
```